modules/getting-started/pages/glossary.adoc

```diff
@@ -0,0 +1,203 @@
+
+mocking service::
+Provides a public URI to an API being designed. Exercising the URI returns the responses defined in the API specification for testing or exploring an API.
+
+Mule® application::
+An app that is configured to run in Mule runtime engine.
+
+module::
+A construct that groups together components. Modules add flexibility to your apps by allowing you to aggregate values, compress data, use Java features, and use extra features for processing JSON.
+
+Mule Enterprise Edition (EE)::
+The enterprise version of Mule, available for 30-day trial download. The Enterprise Edition includes full development cycles, testing, technical support, maintenance releases and hot fixes, and management and monitoring tools from MuleSoft. If you are deploying Mule in a mission-critical environment, want to ensure that you always have a stable, high-quality release, and want additional tools for managing and monitoring your deployment, you should purchase a subscription to Mule Enterprise Edition.
+
+Mule® runtime engine (Mule® or Mule)::
+Java-based integration runtime engine for Anypoint Platform. uses a staged event-driven architecture (SEDA) to enqueue messages and process them inside of flows in separate stages. Mule is the engine for Anypoint Platform™ and the only runtime that combines data and application integration across legacy systems, SaaS applications, and APIs.
+
+= O
+
+organization::
+The container for everything in your Anypoint Platform account. 
+
+= P
+
+Pivotal Cloud Foundry::
+A cloud computing platform as a service (PaaS) provided by the company Pivotal. Anypoint Platform integrates with Pivotal Cloud Foundry, allowing you to deploy Mule applications to dynamically created virtual machines on your own private network. See deployment strategies. Previously referred to as PCF.
+
+policy::
+An object that controls authentication, access, consumption rates, and service level access (SLA) to API resources. Anypoint Platform provides both preconfigured and tools to create custom policies.
+
+= R
+
+RAML®::
+RESTful API Modeling Language (RAML) provides a specification language that you can use to define an API.
+
+Runtime Manager agent::
+The Runtime Manager agent mediates communication between the Anypoint Runtime Manager console and the Mule instances running on servers. Using the Mule agent, you can monitor and control Mule servers. Previously referred to as Runtime agent.
+
+runtime plane::
+The contract between the control plane and data plane for runtime control. In Anypoint Platform, the runtime plane consists of Mule runtime engine and runtime services managed with tools like Anypoint Runtime Fabric.
+
+= S
+
+secrets manager::
+Construct to store and control access to private keys, passwords, certificates, and other secrets.
```

Comment:
I would use service maybe instead of "Construct." Just because construct can be a verb and that's how I first read it.